src/main/java/edu/cornell/kfs/module/ld/businessobject/PositionDataWorkday.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.module.ld.businessobject;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+import org.kuali.kfs.sys.KFSConstants;
+
+import edu.cornell.kfs.module.ld.CuLaborPropertyConstants;
+import edu.cornell.kfs.sys.CUKFSConstants;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+
+public class PositionDataWorkday extends PersistableBusinessObjectBase {
+
+    private String positionNumber;
+    private String jobCode;
+    private Date effectiveDate;
+    private String positionEffectiveStatus;
+    private String description;
+    private String shortDescription;
+    private String businessUnit;
+    private String departmentId;
+    private String positionStatus;
+    private Date statusDate;
+    private String budgetedPosition;
+    private BigDecimal standardHoursDefault;
+    private String standardHoursFrequency;
+    private String positionRegularTemporary;
+    private BigDecimal positionFullTimeEquivalency;
+    private String positionSalaryPlanDefault;
+    private String positionGradeDefault;
+    private Date inactivationDate;
+
+    public PositionDataWorkday() {
+    }
+
+    public String toString() {
+        SimpleDateFormat sdf = new SimpleDateFormat(KFSConstants.MONTH_DAY_YEAR_DATE_FORMAT);
```

Comment:
I am wondering if you could use reflection to output the String value of this object.

Reply:
I will investigate.